src/test/mochitest/browser_dbg-outline.js

```diff
@@ -9,6 +9,25 @@ function getNthItem(dbg, index) {
   return findElement(dbg, "outlineItem", index);
 }
 
+// Tests that the length of outline functions for original and pretty printed source matches
+add_task(async function () {
+  const dbg = await initDebugger("doc-scripts.html");
+  const {
+    selectors: { getSelectedSource },
+    getState
+  } = dbg;
+
+  await selectSource(dbg, "simple1");
+  findElementWithSelector(dbg, ".outline-tab").click();
+  const originalSource = getItems(dbg);
+
+  clickElement(dbg, "prettyPrintButton");
+  await waitForSource(dbg, "simple1:formatted");
+  const prettySource = getItems(dbg);
+
+  is(originalSource.length, prettySource.length, "Length of outline functions for both prettyPrint and originalSource same");
```

Comment:
The test fails for me here but it's a timing issue.  Use `waitForElement` to wait for the function list to be present.  The timing issue is that the pane starts out with content `Loading...`, and so that list wont be there immediately.